BackgroundJobs/BackgroundExchangeService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Corzbank.Data.Entities.Models;
+using Corzbank.Services.Interfaces;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace BackgroundJobs
+{
+    public class BackgroundExchangeService : BackgroundService
+    {
+        private readonly ILogger<BackgroundExchangeService> _logger;
+        private readonly IServiceProvider _serviceProvider;
+        private readonly BackgroundJobModel _backgroundModel;
+
+        public BackgroundExchangeService(ILogger<BackgroundExchangeService> logger, IServiceProvider serviceProvider, BackgroundJobModel backgroundModel)
+        {
+            _logger = logger;
+            _serviceProvider = serviceProvider;
+            _backgroundModel = backgroundModel;
+        }
+
+        protected override async Task ExecuteAsync(CancellationToken stoppingToken)
```

Comment:
Rename **stoppingToken** to **cancellationToken**

Reply:
resolved